### PR TITLE
[ready] add location to game

### PR DESCRIPTION
**Pull request by fatboypunk**
solves #21

it adds a location when a game exists. 
we could make it smarter, like only adding a location after a command `/location` has been set

### CODE REVIEW

File: app/models/location.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#  longitude  :string
+#  latitude   :string
+#  name       :string
+#  created_at :datetime
+#  updated_at :datetime
+#
+
+class Location < ActiveRecord::Base
+  belongs_to :game
+
```
Comment (jurre) [inline app/models/location.rb]: :dog: unnecessary newline

File: spec/support/helpers/telegram_helper.rb [hunk 2]
```diff
@@ -10,14 +10,18 @@ def chat_params
...
+    message_params = {
       chat: telegram_chat,
       from: telegram_user,
       text: text,
       date: 1461013375,
       message_id: 92
     }
+    opts.each do |key, value|
+      message_params[key] = value
+    end
```
Comment (jurre) [inline spec/support/helpers/telegram_helper.rb]: Isn't this the same as:

```ruby
message_params.merge(opts)
```

?

Comment (fatboypunk) [inline spec/support/helpers/telegram_helper.rb]: Yes

**Comment by jurre** [comment]
[LGTM](http://lgtm.in/i/Qegg5iK6q)

LGTM!

**Comment by dkhgh** [comment]
needs a rebase now, sry

**Pull request merged**